### PR TITLE
Dev/adding analytics

### PR DESCRIPTION
**Pull request by Flavius-The-Person**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: "A collection of tips and advice for streamers by the StreamSounds
 logo: 'assets/images/logo.png'
 favicon: 'assets/images/logo.png'
 baseurl: ''
-google_analytics: ''
+#google_analytics: ''
```
Comment (ogjunkyard) [inline _config.yml]: If we aren't using this, then let's remove it completely.

File: _config.yml [hunk 2]
```diff
@@ -34,12 +34,21 @@ authors:
     description: "Co-Founder of The Library by Stream Sounds"
 
 # Plugins
+whitelist: [jekyll-analytics]
```
Comment (ogjunkyard) [inline _config.yml:36]: What does whitelisting do?

File: assets/js/plausible.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class Plausible
+    def initialize(config)
+        @domain = config['domain']
+        @source = config['source']
+    end
+
+    def render()
+        return "<script async defer data-domain=\"#{@domain}\" src=\"#{@source}\"></script>"
```
Comment (ogjunkyard) [inline assets/js/plausible.rb:8]: Plausible.js is mentioned in the _config.yml file as being served by us, but I don't see the javascript file actually in our assets. Or, does this generate the javascript file somehow?

File: Gemfile.lock [hunk 4]
```diff
@@ -60,23 +61,26 @@ GEM
...
       ffi (~> 1.9)
-    terminal-table (1.8.0)
+    terminal-table (2.0.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     unicode-display_width (1.7.0)
+    wdm (0.1.1)
 
 PLATFORMS
-  ruby
+  x64-mingw32
```
Comment (ogjunkyard) [inline Gemfile.lock]: x64-mingw32 shouldn't have been added and ruby shouldn't have been removed.

File: Gemfile.lock [hunk 5]
```diff
@@ -60,23 +61,26 @@ GEM
     rexml (3.2.4)
     rouge (3.26.0)
     safe_yaml (1.0.5)
-    sassc (2.4.0)
+    sassc (2.4.0-x64-mingw32)
```
Comment (ogjunkyard) [inline Gemfile.lock]: drop the -x64-mingw32 from this.

File: Gemfile.lock [hunk 6]
```diff
@@ -8,27 +8,28 @@ GEM
     em-websocket (0.5.2)
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.6.0)
-    eventmachine (1.2.7)
-    ffi (1.13.1)
+    eventmachine (1.2.7-x64-mingw32)
+    ffi (1.14.2-x64-mingw32)
```
Comment (ogjunkyard) [inline Gemfile.lock]: Drop -x64-mingw32 from these.

File: _config.yml (hunk 2 above)
Comment (Flavius-The-Person) [reply _config.yml:36]: by default some plugins are blocked by github pages, whitelisting allows the plugins to be used. That is what was gathered through some of the issues in the plugin repo.

File: assets/js/plausible.rb (hunk 3 above)
Comment (Flavius-The-Person) [reply assets/js/plausible.rb:8]: This PR should be adding plausible.js to assets/js/

File: _config.yml (hunk 1 above)
Comment (Flavius-The-Person) [reply _config.yml]: I will wait and do that in another branch, along with discuss, mailchimp and any other properties we aren't using.

File: assets/js/plausible.rb (hunk 3 above)
Comment (ogjunkyard) [reply assets/js/plausible.rb:8]: Right now, it looks like the source is coming from us per the config file, but I don't see anything in the repo that would actually host that file. If we merge this PR, we may end up getting a 404. If we do get a 404, try changing the source in the config file back to plausible and see if that fixes it.

**Pull request merged**